### ISSUE TITLE
make error on 6.5.0-kali1-arm64 in UTM

### ISSUE THREAD

**Issue by skyper-cz** [aircrack-ng/rtl8188eus#270]
Hello my friend
I wnated to try wifite after a while but my machine changed the driver for the original one. So I went to try recompile it again however I got this arror along the way.

<img width="572" alt="Screenshot 2023-10-27 at 14 43 35" src="https://github.com/aircrack-ng/rtl8188eus/assets/62872636/3f37605b-50ea-4a30-b592-f4d4524ef35c">

Is there solution any at this moment?

**Comment by SmillerMP** [comment]
You are using raspberry pi?

**Comment by skyper-cz** [comment]
No, im using UTM for virtualization, but it was working before but not now.

**Comment by gglluukk** [comment]
driver `rtl8xxxu` coming with newer kernels supports monitor mode so use it before trying this one

otherwise to compile this driver you need to fully update/upgrade your debian-based distro:
`apt update && apt -y upgrade && apt clean && apt autoremove && update-grub && reboot`

after that:
```
apt --yes install build-essential linux-headers-`uname -r`
```
and compile this driver
